### DISCUSSION TITLE
公司掌控了Chrome浏览器的插件管理功能，求教下载了release note里面最新版本的Chrome zip包，如何安装？感谢大佬们求教

### DISCUSSION THREAD

**Discussion by Supwemawill** [immersive-translate/immersive-translate#853] [Help] [answered]
### 概述 | Summary

公司掌控了Chrome浏览器的插件管理功能，求教下载了release note里面最新版本的Chrome zip包，如何安装？感谢大佬们求教

**Comment by versun** [comment, marked as answer]
这种问题谷歌一下就很多哈
管理扩展程序->打开开发者模式->加载
![image](https://github.com/immersive-translate/immersive-translate/assets/2398708/dcc79627-f31f-4af4-8ef9-ba5c2c1cfb20)

**Reply by Supwemawill** [reply]
公司IT连加载已解压的拓展程序按钮都block了……

**Reply by versun** [reply]
那我们也没办法....抱歉

**Comment by keyboard3** [comment]
这么狠嘛….

**Reply by Supwemawill** [reply]
害 一言难尽，毕竟公司对于数据和信息安全也有比较高的要求，没办法了
